--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -44,16 +44,6 @@
 )
 
 bazel_dep(
-    name = "rules_jvm_external",
-    version = "6.3",
-)
-
-bazel_dep(
-    name = "rules_kotlin",
-    version = "1.9.6",
-)
-
-bazel_dep(
     name = "rules_license",
     version = "1.0.0",
 )
@@ -128,31 +118,6 @@
 
 use_repo(pip, "pip_deps")
 
-maven = use_extension("@rules_jvm_external//:extensions.bzl", "maven")
-
-maven.install(
-    artifacts = [
-        "com.google.caliper:caliper:1.0-beta-3",
-        "com.google.code.findbugs:jsr305:3.0.2",
-        "com.google.code.gson:gson:2.8.9",
-        "com.google.errorprone:error_prone_annotations:2.5.1",
-        "com.google.j2objc:j2objc-annotations:2.8",
-        "com.google.guava:guava:32.0.1-jre",
-        "com.google.guava:guava-testlib:32.0.1-jre",
-        "com.google.truth:truth:1.1.2",
-        "junit:junit:4.13.2",
-        "org.mockito:mockito-core:4.3.1",
-        "biz.aQute.bnd:biz.aQute.bndlib:6.4.0",
-        "info.picocli:picocli:4.6.3",
-    ],
-    repositories = [
-        "https://repo1.maven.org/maven2",
-        "https://repo.maven.apache.org/maven2",
-    ],
-)
-
-use_repo(maven, "maven")
-
 # Development dependencies
 bazel_dep(
     name = "googletest",
